packages/viewer/src/lib/interactions/interactionApi.ts

```diff
@@ -370,25 +370,25 @@ export abstract class InteractionApi<T extends Camera = Camera> {
    *  viewer.
    */
   public async rotateCamera(delta: Point.Point): Promise<void> {
-    return this.transformCamera(({ camera, viewport }) => {
+    return this.transformCamera(({ camera, viewport, boundingBox }) => {
       const upVector = Vector3.normalize(camera.up);
-      const lookAt = Vector3.normalize(
+      const directionVector = Vector3.normalize(
         Vector3.subtract(camera.lookAt, camera.position)
       );
-
-      const crossX = Vector3.cross(upVector, lookAt);
-      const crossY = Vector3.cross(lookAt, crossX);
+      const crossX = Vector3.cross(upVector, directionVector);
+      const crossY = Vector3.cross(directionVector, crossX);
 
       const mouseToWorld = Vector3.normalize({
         x: delta.x * crossX.x + delta.y * crossY.x,
         y: delta.x * crossX.y + delta.y * crossY.y,
         z: delta.x * crossX.z + delta.y * crossY.z,
       });
 
-      const rotationAxis = Vector3.cross(mouseToWorld, lookAt);
+      const rotationAxis = Vector3.cross(mouseToWorld, directionVector);
 
-      const epsilonX = (3.0 * Math.PI * delta.x) / viewport.width;
-      const epsilonY = (3.0 * Math.PI * delta.y) / viewport.height;
+      // The 9.5 multiplier was chosen to match the desired rotation speed
```

Comment:
Since we're also multiplying by 3 (to set the desired rotation speed), there's no reason to multiply by PI. We can multiply by 9.5 instead, which will have essentially the same results and be slightly faster.